try:
    
    numero = int(input("Introducí un valor numérico: "))
    print(f"el valor es el {numero}")
  
except Exception as ex:
    print(f"ocurrrió un error")
       

